domains/Discussions/Tests/Feature/AnswersIndexControllerTest.php

```diff
@@ -0,0 +1,174 @@
+use Domains\Discussions\Models\Question;
+use Laravel\Lumen\Testing\DatabaseMigrations;
+use Domains\Accounts\Database\Factories\UserFactory;
+use Domains\Discussions\Database\Factories\AnswerFactory;
+use Domains\Discussions\Database\Factories\QuestionFactory;
+
+class AnswersIndexControllerTest extends TestCase
+{
+    use DatabaseMigrations;
+
+    private User $user;
+    private User $secondUser;
+    private Question $question;
+    private Answer $answer;
+    private Answer $secondAnswer;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->user = UserFactory::new()->create();
+        $this->secondUser = UserFactory::new()->create();
+
+        $this->question = QuestionFactory::new([
+            'author_id' => $this->user->id
+        ])->create();
+
+        $this->answer = AnswerFactory::new([
+            'question_id' => $this->question->id,
+            'author_id' => $this->user->id,
+            'created_at' => Carbon::now()->subWeek()->toDateTimeString()
+        ])->create();
+
+        $this->secondAnswer = AnswerFactory::new([
+            'question_id' => $this->question->id,
+            'author_id' => $this->secondUser->id,
+            'created_at' => Carbon::now()->toDateTimeString()
+        ])->create();
+    }
+
+    /** @test */
+    public function it_gets_paginated_answers_for_a_question(): void
+    {
+        $this->get(route('discussions.questions.answersList', ['questionId' => $this->question->id]))
+            ->seeJsonStructure([
+                "data" => [
+                    0 => [
+                        "id",
+                        "content",
+                        "question" => [
+                            "id",
+                            "title",
+                            "slug",
+                            "description",
+                            "author" => [
+                                "id",
+                                "name",
+                                "email",
+                                "trusted",
+                                "created_at",
+                                "updated_at",
+                                "deleted_at",
+                            ],
+                            "created_at",
+                            "updated_at",
+                            "resolved_at",
+                            "deleted_at",
+                        ],
+                        "author" => [
+                            "id",
+                            "name",
+                            "email",
+                            "trusted",
+                            "created_at",
+                            "updated_at",
+                            "deleted_at",
+                        ],
+                        "created_at",
+                        "updated_at",
+                        "deleted_at",
+                    ]
+                ]
+            ])
+            // answer 1
```

Comment:
Nit: the comment doesn't give any more context than the code itself. We can remove this, if you want.